core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -1037,4 +1037,52 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
             JSON.stringify({ keyword, languageId, archived })
         );
     }
+
+    /**
+     * Get user permission for a page or contentlet
+     *
+     * @private
+     * @param {DotCMSContentlet} item
+     * @return {*}  {UserPagePermission}
+     * @memberof DotPageStore
+     */
+    private getUserPagePermissions(item: DotCMSContentlet): UserPagePermission {
+        // Logged user
+        const { loggedUser } = this.get();
+
+        // Item types
+        const isPage = item.baseType === 'HTMLPAGE';
```

Comment:
@rjvelazco should we create a constant for `CONTENT` and `HTMLPAGE`?

Reply:
Could be a good idea, idk.

Reply:
Done!

Reply:
we have `DotCMSBaseTypesContentTypes`

Reply:
Done!